tsconfig.json

```diff
@@ -14,10 +16,9 @@
     "suppressImplicitAnyIndexErrors": true,
     "strictNullChecks": true,
     "noFallthroughCasesInSwitch": true,
-    "noUnusedParameters": true,
+    "noUnusedParameters": false,
     "noUnusedLocals": true
   },
-  "include": ["src", "typings"],
```

Comment:
Why have you removed this? I store a lot of extra stuff (tests for user issues, WIP code, local test scripts, etc) that I wouldn't want to be affected by tsconfig

Reply:
mmm.  i think i see what you mean.  flipping tsconfig to `allowJs: false` prevents `.js` files from being copied into the build, so those files would be omitted anyway if we auto gen the declarations.  if that content needs to ship, we can do it in a post processing step.  does that work?  are you certain those need to be shipped?